CHANGES.md

```diff
@@ -101,8 +102,6 @@ Internal Changes
 - Remove dead code from supporting ACME. ([\#11393](https://github.com/matrix-org/synapse/issues/11393))
 - Refactor including the bundled relations when serializing an event. ([\#11408](https://github.com/matrix-org/synapse/issues/11408))
 
-
-
```

Comment:
nit: The convention seems to be 2 blank lines between sections, judging by the changelog entries below